src/site/_includes/js/eligibility.js

```diff
@@ -457,7 +457,11 @@ function updateIncomeTotal() {
   for (input of inputs) {
     sum += Number(input.value);
   }
-  totalDisplay.textContent = sum;
+  sum = parseFloat(sum).toFixed(2)
```

Comment:
`sum` should already be a `Number` thanks to the conversion done in the summation loop.  So I think `sum = sum.toFixed(2);` should work ok.